src/rust/src/x509/verify.rs

```diff
@@ -185,9 +188,69 @@ impl PyStore {
     }
 }
 
+#[pyo3::pyclass(
+    frozen,
+    name = "Policy",
+    module = "cryptography.hazmat.bindings._rust.x509"
+)]
+struct PyPolicy(OwnedPolicy);
+
+impl PyPolicy {
+    fn as_policy(&self) -> &Policy<'_, PyCryptoOps> {
+        &self.0.borrow_dependent().0
+    }
+}
+
+#[pyo3::prelude::pyfunction]
+fn verify<'p>(
+    py: pyo3::Python<'p>,
+    leaf: &PyCertificate,
+    policy: &PyPolicy,
+    intermediates: &'p pyo3::types::PyList,
+    store: &'p PyStore,
+) -> CryptographyResult<Vec<PyCertificate>> {
```

Comment:
TODO: This now needs to become an instance method on `PyServerVerifier`. That will eliminate the use of `PyPolicy`, which has been replaced by `PyServerVerifier` as well.